backend/api/Controllers/RobotController.cs

```diff
@@ -637,5 +638,81 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
             return Ok(missionRun);
         }
 
+        /// <summary>
+        ///   Empties the mission queue for the robot, stops the ongoing mission, sets the robot to available and current area is set to null
+        /// </summary>
+        /// <remarks>
+        ///     <para> This query resets the robot </para>
+        /// </remarks>
+        [HttpPut]
+        [Authorize(Roles = Role.Admin)]
+        [Route("{robotId}/reset-robot")]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        public async Task<ActionResult> ResetRobot(
+            [FromRoute] string robotId
+        )
+        {
+            Console.WriteLine(robotId);
+            Console.WriteLine("oooooooooo....................");
```

Comment:
Don't think these are load-bearing printouts so they can probably be deleted

Reply:
Damn can't believe i forgot these